crates/sncast/src/starknet_commands/account/import.rs

```diff
@@ -196,9 +201,57 @@ fn get_private_key_from_input() -> Result<Felt> {
     parse_input_to_felt(&input)
 }
 
+pub fn get_import_name(
+    import_name: Option<String>,
+    accounts_file: &Utf8PathBuf,
+) -> (String, Option<u32>) {
+    if let Some(account) = import_name {
+        (account, None)
+    } else {
+        if !accounts_file.exists() {
+            std::fs::create_dir_all(accounts_file.clone().parent().unwrap())
+                .expect("Failed to create directory");
+            std::fs::write(accounts_file.clone(), "{}").expect("Failed to write to file");
+        }
+
+        let contents = std::fs::read_to_string(accounts_file.clone()).expect("Failed to read file");
+        let mut items: serde_json::Value = serde_json::from_str(&contents)
+            .map_err(|_| anyhow!("Failed to parse accounts file at = {}", accounts_file))
+            .unwrap();
+
+        let mut available_default_ids: Vec<u32> = serde_json::from_value(
+            items["default_account_settings"]["available_default_ids"].clone(),
+        )
+        .unwrap_or(vec![]);
+        let total_defaults: u32 =
+            serde_json::from_value(items["default_account_settings"]["total_defaults"].clone())
+                .unwrap_or(0);
+
+        let total_defaults = total_defaults + 1;
+        let default_id = if available_default_ids.is_empty() {
+            total_defaults
+        } else {
+            let min_id = *available_default_ids.iter().min().unwrap();
+            available_default_ids.retain(|x| *x != min_id);
+            min_id
+        };
+        items["default_account_settings"]["available_default_ids"] = json!(available_default_ids);
+        items["default_account_settings"]["total_defaults"] = json!(total_defaults);
+
+        std::fs::write(
+            accounts_file.clone(),
+            serde_json::to_string_pretty(&items).unwrap(),
+        )
+        .expect("Failed to write to file");
+
+        (format!("account-{default_id}"), Some(default_id))
+    }
+}
+
```

Comment:
The idea of calculating newly added account index should be simplified, there's no need to store information about total ids etc. 

The idea could look as follows:
- read all accounts with name matching `account-{i}` pattern
- extract their numbers
- find the first free smallest number which will be then used for new account name

Is that idea clear to you?